core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -945,11 +945,38 @@ class HttpGoogleServicesDAO(
     } yield updated
   }
 
-  override def deleteProject(googleProject: GoogleProjectId): Future[Unit]= {
+  // TODO - once workspace migration is complete and there are no more v1 workspaces or v1 billing projects, we can remove this https://broadworkbench.atlassian.net/browse/CA-1118
+  // V2 workspace projects are managed by rawls SA but the v1 billing google projects are managed by the billing SA.
+  override def deleteV1Project(googleProject: GoogleProjectId): Future[Unit]= {
```

Comment:
looks like github struggled a bit with the diff here.

The changes listed here were implemented so that it would be simpler and cleaner to rip out v1 billing google project deletion once migration is complete:
- deleteProject was renamed to deleteV1Project. this is used by v1 billing, to delete the google project. Those google projects cannot be managed by the rawls SA.
- a copy of deleteProject was made (to be used moving forward, with v2 workspaces, which uses the rawls SA creds.